# HG changeset patch
# User "Cole Robinson <crobinso@redhat.com>"
# Date 1224708808 14400
# Node ID c44f1d9698091da7ce8e368aba30c68b580b5c10
# Parent  e3c76e40eb2e5d468c8ff61f87f1dd87ef903120
Only popup vol copy path menu on right click (not any click)

diff -r e3c76e40eb2e -r c44f1d969809 src/virtManager/host.py
--- a/src/virtManager/host.py	Wed Oct 22 16:48:15 2008 -0400
+++ b/src/virtManager/host.py	Wed Oct 22 16:53:28 2008 -0400
@@ -614,6 +614,9 @@
         self.window.get_widget("vol-delete").set_sensitive(True)
 
     def popup_vol_menu(self, widget, event):
+        if event.button != 3:
+            return
+
         tup = widget.get_path_at_pos(int(event.x), int(event.y))
         if tup == None:
             return False
